wiki/HistoricalTimeline.md

```diff
@@ -136,10 +156,16 @@ This timeline is provided mainly to assist with story writing, to make sure that
 
 * 2920: The spread of new colonies has stretched the Republic Navy thin, and piracy is once again on the rise in the corners of Syndicate space and on the Rim.
 
+* 2940: For the first time, large warships are made available for private use, as merchant fleets increasingly rely on their own defensive forces rather than the Navy. 
+
 * 2950: The space renaissance has spurred the creation of many other manufacturing companies, including Delta V, Tarazed, Betelgeuse Shipyards, and Megaparsec. This leads to increased creativity and variety in starship design.
 
+* 2970: The Republic Parliament passes the Galactic Human Rights Charter, outlawing slavery across the galaxy. Several wealthy families on the Paradise planets refuse to emancipate their slaves, resulting in a temporary occupation of Glory by the Navy.
+
 * 2980: Many Dirt Belt worlds are growing in population, but many of those worlds are barely habitable and too poor to afford terraforming. The gap between rich and poor worlds is growing.
 
+* 2990: Lionheart begins production of a modified colonization ship, intended for both commerce and convoy defense.
```

Comment:
```suggestion
```

Reply:
I understand this is supposed to be alluding to the Bactrian, but the Bactrian dates back to the era of City-Ships, and we know that era was over before the Alpha Wars, since Firebirds were used in those wars.

What I would maybe suggest is instead having an entry at some time in the 2300s along the lines of 
`Advances in technology make smaller starships economically feasible for the first time, bringing the era of city ships to an abrupt end. The newly established Betelgeuse Shipyards quickly becomes a major player, with a range of small cutting-edge ships available at low prices which would have been unthinkable a decade prior.`